apps/mech/common-util/functions/graphql.ts

```diff
@@ -55,7 +61,15 @@ type FetchMmMechsArgs = {
   };
 };
 
-export async function fetchMmMechs({ first, total, filters }: FetchMmMechsArgs) {
+export type MmMech = {
+  id: number;
+  address: string;
+  mechFactory: string;
+  configHash: string;
+  owner: string;
+};
+
+export async function fetchMmMechs({ first, total, filters }: FetchMmMechsArgs): Promise<MmMech[] | Error> {
```

Comment:
i assume this is supposed to be `fetchMyMechs`, let me know if correct, will update it.